frontend/public/reducers/ui.ts

```diff
@@ -362,3 +362,9 @@ export const getActiveNamespace = ({ UI }: RootState): string => UI.get('activeN
 export const getActivePerspective = ({ UI }: RootState): string => UI.get('activePerspective');
 
 export const getActiveApplication = ({ UI }: RootState): string => UI.get('activeApplication');
+
+export const getSelectedTimespan = ({ UI }: RootState): number =>
+  UI.getIn(['monitoringDashboards', 'timespan']);
+
+export const getSelectedPollInterval = ({ UI }: RootState): number =>
+  UI.getIn(['monitoringDashboards', 'pollInterval']);
```

Comment:
nit: as it's used just once here , can fetch it directly in connected component.